pages/services/edge-lb/1.2/uninstalling/index.md

```diff
@@ -1,18 +1,23 @@
 ---
 layout: layout.pug
-navigationTitle:  Uninstalling 
-title: Uninstalling 
+navigationTitle:  Uninstalling
+title: Uninstalling
 menuWeight: 20
 excerpt: Uninstalling the Edge-LB package
 
 enterprise: false
 ---
 
 
-<p class="message--warning"><strong>WARNING: </strong>Do not use the GUI or CLI (Marathon) to create or destroy services managed by Edge-LB. Operations must be performed exclusively through the Edge-LB CLI.</p>
+<p class="message--warning"><strong>WARNING: Do not use the GUI or CLI (Marathon) to create or destroy services managed by Edge-LB. Operations must be performed exclusively through the Edge-LB CLI or the Edge-LB API.</p>
```

Comment:
I think we never say GUI for the UI. Maybe?

`Do not use the Marathon UI or the Marathon CLI...`

or even, if talking about DC/OS UI:

`Do not use the DC/OS UI or the Marathon CLI...`